src/test/java/seedu/address/testutil/TypicalTasks.java

```diff
@@ -17,26 +19,26 @@
 public class TypicalTasks {
 
     public static final Task TASK_1 = new TaskBuilder().withDescription("Todo 2101")
-            .withDeadline("2022-01-11").build();
+            .withDeadline("2022-01-11").withMark("false").build();
     public static final Task TASK_2 = new TaskBuilder().withDescription("Todo 2102")
-            .withDeadline("2022-02-16").build();
+            .withDeadline("2022-02-16").withMark("true").build();
     public static final Task TASK_3 = new TaskBuilder().withDescription("Todo 2103")
-            .withDeadline("2022-03-15").build();
+            .withDeadline("2022-03-15").withMark("false").build();
     public static final Task TASK_4 = new TaskBuilder().withDescription("Todo 2104")
-            .withDeadline("2022-04-14").build();
+            .withDeadline("2022-04-14").withMark("true").build();
     public static final Task TASK_5 = new TaskBuilder().withDescription("Todo 2105")
-            .withDeadline("2022-05-13").build();
+            .withDeadline("2022-05-13").withMark("false").build();
     public static final Task TASK_6 = new TaskBuilder().withDescription("Todo 2106")
-            .withDeadline("2022-06-12").build();
+            .withDeadline("2022-06-12").withMark("true").build();
     public static final Task TASK_7 = new TaskBuilder().withDescription("Todo 2107")
-            .withDeadline("2022-07-11").build();
+            .withDeadline("2022-07-11").withMark("false").build();
 
```

Comment:
Might want to shuffle dates here as well for easier testing.

Reply:
Same as previous, order has been changed. Need to change the json file typicalProjects for the test files too